packages/sdk/src/config/README.md

```diff
@@ -5,72 +5,69 @@ This is the method to setup the global variables object on all your mintbase-js
 
 You can set them in three ways:
 
+### 1. Use the config method (preferred)
+If you want to setup on our config method, simply call the method with your config settings in the main file of your project.
 
-- 1. if you want to setup on our config method, in the main file of your project.This method will work on both server side and client side.
-note: thats the most advisable way to work with global variables on the mb.js packages.
+This method will work on both server side and client side.
 
-  ```typescript
-    import { mbjs } from '@mintbase-js/sdk';
+```typescript
+  import { mbjs } from '@mintbase-js/sdk';
 
-     const config = {
-          network: 'testnet',
-          callbackUrl: 'https://mintbase.xyz/success',
-          contractAddress: 'buddha.mintspace2.testnet'
-      }
-
-    mbjs.config(config)
-
-    //can retrieve the keys in any part of your application.
-    console.log(mbjs.keys , 'global keys of all mintbase-js packages')
-  ```
+    const config = {
+        network: 'testnet',
+        callbackUrl: 'https://mintbase.xyz/success',
+        contractAddress: 'buddha.mintspace2.testnet'
+    }
 
+  mbjs.config(config)
 
-- 2. straight on the method itself if it accepts the variable as an argument.
+  //can retrieve the keys in any part of your application.
+  console.log(mbjs.keys , 'global keys of all mintbase-js packages')
+```
 
-  ```typescript
-    import { tokenById } from '@mintbase-js/data';
 
-    const { data, error } = await tokenById('5','logout.mintbase1.near', 'mainnet');
-  ```
+### 2. Pass the network directly to methods (where applicable).
```

Comment:
contractAddress is one env variable that also is supported in some methods ( and also can be set on mbjs.config )

Reply:
Let's address some of this in the testing task we have in the sprint. Restore integration tests as well as get some unit testing around using keys inside of the methods. I just had to fix a bug while testing HERE wallet

Reply:
Can you un-request these changes and we can add in a second pass? Just trying to ship the HERE wallet now